tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md

```diff
@@ -75,7 +75,7 @@ final List<MyBusinessPartnerType> businessPartners = ODataQueryBuilder
                 "IsFemale",
                 "CreationDate")
         .build()
-        .execute(destination)
+        .executeRequest(destination)
```

Comment:
This change is actually not correct. The `ODataQueryBuilder` is not part of the Cloud SDK and to my knowledge hasn't changed. 

It's not critical because this code is not meant to be run by the reader, but still we should provide valid code. So let's revert this.